core/src/main/java/org/eclipse/hono/util/MessageHelper.java

```diff
@@ -100,6 +100,10 @@
      * reported the data belongs to.
      */
     public static final String APP_PROPERTY_TENANT_ID = "tenant_id";
+    /**
+     * The name of the AMQP 1.0 message application property containing the resource version expected by the client.
+     */
+    public static final String APP_PROPERTY_RESOURCE_VERSION = "resource_version";
```

Comment:
do we actually ever have such an application property in an AMQP message anywhere?

Reply:
It is used in the `EventBusMessage` and all constants there seem to come from the class `org.eclipse.hono.util.MessageHelper`. So it seems logical to add this there as well.